LayoutTests/interaction-region/split-inline-link-expected.txt

```diff
@@ -0,0 +1,28 @@
+A bunch of text before the first line
+short second line.
+(GraphicsLayer
+  (anchor 0.00 0.00)
+  (bounds 800.00 600.00)
+  (children 1
+    (GraphicsLayer
+      (bounds 800.00 600.00)
+      (contentsOpaque 1)
+      (drawsContent 1)
+      (backgroundColor #FFFFFF)
+      (event region
+        (rect (0,0) width=800 height=600)
+
+      (interaction regions [
+        (region
+            (rect (197,-3) width=31 height=18)
+            (rect (-3,15) width=115 height=6)
```

Comment:
These rects are kind of bad but I'll look into progressing them in a future patch.